lab_1_classify_by_unigrams/main.py

```diff
@@ -19,6 +26,13 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
+    if not isinstance(tokens, list):
+        return None
+    for token in tokens:
+        if not isinstance(token, str):
+            return None
+    my_dict = {token: tokens.count(token) / len(tokens) for token in tokens}
```

Comment:
do not create a function, return directly

Reply:
fixed